docs/source/notes/libtorch_stable_abi.md

```diff
@@ -9,6 +9,7 @@ This note will eventually contain more details on how to use the APIs in torch/c
 |  type in custom extension    |   StableIValue representation   |   type in libtorch  |   Schema Type  |
 | -------- | ------- | ------- | ------- |
 | std::optional\<S> | raw bitwise copy into leading bytes of uint64_t of pointer to a new StableIValue representing S | std::optional\<T> | Type? |
+| std::optional\<S>() | nullptr | std::optional\<T>() | Type? |
 | std::nullopt | nullptr | IValue() | None |
```

Comment:
The third column here doesn't track for me -- isn't it IValue(std::optional\<T>()), [which is equivalent to IValue()](https://github.com/pytorch/pytorch/blob/main/aten/src/ATen/core/ivalue_inl.h#L2308)? I would expect lines 12 and 13 here to be redundant (and I guess I would advise removing line 13).